pkg/language/checks.go

```diff
@@ -391,24 +391,40 @@ func (r *Reader) checkPost(method *concepts.Method) {
 }
 
 func (r *Reader) checkUpdate(method *concepts.Method) {
-	// Only scalar and struct parameters:
+	// Only scalar, struct and list parameters:
 	for _, parameter := range method.Parameters() {
-		if !parameter.Type().IsScalar() && !parameter.Type().IsStruct() {
+		if !parameter.Type().IsScalar() && !parameter.Type().IsStruct() &&
+			!parameter.Type().IsList(){
 			r.reporter.Errorf(
-				"Type of parameter '%s' should be scalar or struct but it is '%s'",
+				"Type of parameter '%s' should be scalar, struct or list but it is '%s'",
 				parameter, parameter.Type().Kind(),
 			)
 		}
 	}
 
+	// Exactly one list parameter:
+	var lists []*concepts.Parameter
+	for _, parameter := range method.Parameters() {
+		if parameter.Type().IsList() {
+			lists = append(lists, parameter)
+		}
+	}
+	count := len(lists)
+	if count != 1 {
```

Comment:
did you mean `count > 1` and "at most one list parameter" ?

Reply:
I copied the same code from func (r *Reader) checkList(method *concepts.Method) {

Reply:
count > 1 is the right check on PATCH list.